opensearch-observability/build.gradle

```diff
@@ -223,9 +223,11 @@ integTest {
     }
 
     // https://github.com/opensearch-project/index-management/pull/93
-    if (System.getProperty("tests.clustername") != null) {
-        exclude 'org/opensearch/observability/ObservabilityPluginIT.class'
-    }
+    // if (System.getProperty("tests.clustername") != null) {
```

Comment:
Do we need to keep this commented out section?

Reply:
i kept it because these are two issues: it fails when running against a remote cluster (same issue in index-management), and it fails under JDK 17 only. This is likely coming from upstream `OpenSearchIntegTestCase` class, keeping these lines in case it got fixed or we want to add more tests under it